src/Symfony/Component/Validator/Constraints/Hostname.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ *
+ * @author Dmitrii Poddubnyi <dpoddubny@gmail.com>
+ */
+class Hostname extends Constraint
+{
+    const INVALID_HOSTNAME_ERROR = '7057ffdb-0af4-4f7e-bd5e-e9acfa6d7a2d';
+
+    protected static $errorNames = [
+        self::INVALID_HOSTNAME_ERROR => 'INVALID_HOSTNAME_ERROR',
+    ];
+
+    public $message = 'This value is not a valid hostname.';
```

Comment:
this message must be added in translation files.

Reply:
I added a couple of translations. How do you fill other translation files?